### PR TITLE
Local hospital advice 2

### PR DESCRIPTION
**Pull request by tobylane**
**Describe what the proposed change does**
- Add advice about adding to a full training room
- Remaining advice about the local hospital is made player only

### CODE REVIEW

**Review by lewri — APPROVED** [review]
I had to read the change to hospital.lua a few times but yes lgtm

File: CorsixTH/Lua/rooms/research.lua [hunk 1]
```diff
@@ -131,12 +131,12 @@ function ResearchRoom:roomFinished()
   -- Is this the first research department built?
   if not self.hospital.research_dep_built and not TheApp.using_demo_files then
     self.hospital.research_dep_built = true
-    self.world.ui.adviser:say(_A.information.initial_general_advice
-    .research_now_available)
+    self.hospital:giveAdvice({ _A.information.initial_general_advice
+    .research_now_available })
```
Comment (Alberth289346) [inline CorsixTH/Lua/rooms/research.lua:135]: Add 2 indents, it's a continuation of the statement.

**Comment by lewri** [comment]
> I don't like the added spaces inside the curly braces, but our code conventions allows it apparently, it's allowed.

I thought that a little weird too. But I'm not actually fussed by it.

**Comment by tobylane** [comment]
We could change the coding convention?

**Comment by lewri** [comment]
While not stated, the nature of brackets used in coding conventions suggest no spacing e.g.
https://github.com/CorsixTH/CorsixTH/wiki/Coding-Conventions#bracketless-function-calls

Ideally then imo, if we were to stick to the norm the spaces would have to go.

**Comment by tobylane** [comment]
Agreed and done, and squashed the autopsy commit into the general one.

**Comment by TheCycoONE** [comment]
Thanks

**Pull request merged**